app/questionnaire/rules.py

```diff
@@ -55,13 +55,34 @@ def evaluate_repeat(repeat_rule, answer_store):
         return repeat_function(filtered)
 
 
-def evaluate_when_rules(when_rules, metadata, answer_store, group_instance=0):
+def evaluate_skip_condition(skip_condition, metadata, answer_store, group_instance=0):
+    """
+    Determine whether a skip condition will be satisfied based on a given answer
+    :param skip_condition: skip_condition rule to evaluate
+    :param metadata: metadata for evaluating rules with metadata conditions
+    :param answer_store: store of answers to evaluate
+    :param group_instance: when evaluating a when rule for a repeating group, defaults to 0 for non-repeating groups
+    :return: True if the when condition has been met otherwise False
+    """
+
+    no_skip_condition = skip_condition is None or len(skip_condition) == 0
```

Comment:
I refactored to be a variable for clarity but happy to change if you don't think it makes things clearer